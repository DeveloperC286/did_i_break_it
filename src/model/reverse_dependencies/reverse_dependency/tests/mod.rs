use super::*;

mod content;

#[test]
fn test_centerline_parse_content_for_reverse_dependencies() {
    // Given
    let content = content::CENTERLINE;

    // When
    let reverse_dependencies = ReverseDependency::from(content);

    // Then
    insta::assert_debug_snapshot!(
        "test_centerline_parse_content_for_reverse_dependencies",
        reverse_dependencies
    );
}

#[test]
fn test_tesseract_sys_parse_content_for_reverse_dependencies() {
    // Given
    let content = content::TESSERACT_SYS;

    // When
    let reverse_dependencies = ReverseDependency::from(content);

    // Then
    insta::assert_debug_snapshot!(
        "test_tesseract_sys_parse_content_for_reverse_dependencies",
        reverse_dependencies
    );
}
