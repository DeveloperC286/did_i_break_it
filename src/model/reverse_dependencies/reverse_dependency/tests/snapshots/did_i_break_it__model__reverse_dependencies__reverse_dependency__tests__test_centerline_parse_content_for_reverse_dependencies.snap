---
source: src/model/reverse_dependencies/reverse_dependency/tests/mod.rs
expression: reverse_dependencies

---
Ok(
    [
        ReverseDependency {
            name: "toxicblend",
            version: "0.0.10",
            version_required: "^0.4.1",
        },
    ],
)
